src/Workspaces/Core/Portable/SymbolKey/SymbolKey.ErrorTypeSymbolKey.cs

```diff
@@ -57,6 +88,28 @@ public static SymbolKeyResolution Resolve(SymbolKeyReader reader)
                 return CreateResolution(result);
             }
 
+            private static SymbolKeyResolution ResolveContainer(SymbolKeyReader reader)
+            {
+                var type = reader.ReadInteger();
+                switch (type)
+                {
+                    case 0:
+                        return reader.ReadSymbolKey();
+                    case 1:
+                        var namespaceNames = reader.ReadStringArray();
+                        var currentNamespace = reader.Compilation.GlobalNamespace;
+
+                        foreach (var name in namespaceNames)
+                            currentNamespace = reader.Compilation.CreateErrorNamespaceSymbol(currentNamespace, name);
+
+                        return new SymbolKeyResolution(currentNamespace);
+                    case 2:
+                        return reader.ReadSymbolKey();
```

Comment:
Can we Debug.Assert() that this didn't really do anything, since it should always be null?

Reply:
good idea